react/README.md

```diff
@@ -624,19 +635,19 @@ We don’t recommend using indexes for keys if the order of items may change.
   - Ordering for `class extends React.Component`:
 
   1. optional `static` methods
-  1. `constructor`
-  1. `getChildContext`
-  1. `componentWillMount`
-  1. `componentDidMount`
-  1. `componentWillReceiveProps`
-  1. `shouldComponentUpdate`
-  1. `componentWillUpdate`
-  1. `componentDidUpdate`
-  1. `componentWillUnmount`
-  1. *clickHandlers or eventHandlers* like `onClickSubmit()` or `onChangeDescription()`
-  1. *getter methods for `render`* like `getSelectReason()` or `getFooterContent()`
-  1. *optional render methods* like `renderNavigation()` or `renderProfilePicture()`
-  1. `render`
+  2. `constructor`
```

Comment:
This change "prettier" does automatically. It looks good to me.
Shall we keep it with this PR? Or delete? Or create a separate PR?

Reply:
Please do not run prettier on any project that doesn't already have it set up.

Reply:
so yes, please revert all unrelated changes, and please do not run prettier on anything.

Reply:
Got it